src/osgPlugins/txp/trpage_readbuf.cpp

```diff
@@ -721,5 +721,4 @@ trpgrAppFileCache::OpenFile::OpenFile()
     id = -1;
     row = 0;
     col = 0;
-
```

Comment:
lets just take the 3.6 version of this file.

Reply:
Reverted changes to this file (pending)